### ISSUE TITLE
Expose HtmlOptions options in WASM

### ISSUE THREAD

**Issue by tnajdek** [zotero/citeproc-rs#115]
It would be great if `HtmlOptions`, especially `link_anchors`, was configurable for `Markup` output from within WASM environment.

**Comment by cormacrelf** [comment]
Good catch, thanks. Turns out this is also required in CSL 1.0.2, not just "would be great".
